lib/App.js

```diff
@@ -45,14 +47,71 @@ const FLOW_TYPES = [ 'triggers', 'conditions', 'actions' ];
 
 class App {
 
-  constructor( appPath ) {
+	constructor( appPath ) {
     this.path = path.resolve(appPath);
+    this.srcPath = this.path;
+    this._app = new HomeyLibApp( this.path );
+    this._appJsonPath = path.join( this.path, 'app.json' );
+    this._pluginsPath = path.join( this.path, '.homeyplugins.json');
```

Comment:
I think you accidentally restored this when resolving a merge conflict, they should be unused.
`this.srcPath` is a bit strange too, it is mutated in preprocess but not used elsewhere.